core/templates/pages/exploration-editor-page/translation-tab/services/translation-tab-active-content-id.service.ts

```
@@ -45,10 +45,13 @@ export class TranslationTabActiveContentIdService {
   }
 
   setActiveContent(contentId: string, dataFormat: string): void {
-    var allContentIds = (
-      this._stateRecordedVoiceoversService.displayed.getAllContentIds());
-    if (allContentIds.indexOf(contentId) === -1) {
-      throw new Error('Invalid active content id: ' + contentId);
+    const displayStateRecordedVoiceovers = (
+      this._stateRecordedVoiceoversService.displayed);
+    if (displayStateRecordedVoiceovers !== null) {
```

What if `displayStateRecordedVoiceovers` is `null`, should we throw an error?

---

Done